README.md

```diff
@@ -120,13 +122,37 @@ InvisibleCaptcha.setup do |config|
   # config.timestamp_threshold = 4
   # config.timestamp_enabled   = true
   # config.injectable_styles   = false
+  # config.ip_enabled          = true
+  # config.secret              = REQUIRED_SECRET_KEY
 
   # Leave these unset if you want to use I18n (see below)
   # config.sentence_for_humans     = 'If you are a human, ignore this field'
   # config.timestamp_error_message = 'Sorry, that was too quick! Please resubmit.'
 end
 ```
 
+#### IP Enabled Setup
+
+View code:
+
+```erb
+<%= form_for(@topic) do |f| %>
+  <%= f.invisible_captcha :subtitle, { timestamp: @invisible_captcha_values.timestamp, spinner_value: @invisible_captcha_values.spinner_value } %>
+  <!-- or -->
+  <%= invisible_captcha :subtitle, :topic, { timestamp: @invisible_captcha_values.timestamp, spinner_value: @invisible_captcha_values.spinner_value } %>
```

Comment:
Ideally this shouldn't be set by the user, I think we can do it automatically under the hood if the `ip_check` is enabled

Reply:
I would agree, but I ran into a problem with getting the helper_method working..

I posted this...  https://stackoverflow.com/questions/66112686/railtie-and-undefined-method-helper-method-whats-the-deal

If you have any ideas, please let me know.

Reply:
Ok! sorry I have very limited time this week, I'll try to take a look during the weekend. But ideally, I'd like to remove all public API related to this new spinner thing. Probably, one thing we can explore is to make all this machinery automatically using the `invisible_captcha` view helper.